### ISSUE TITLE
External search command 'pantag' returned error code 2. Script output = "ERROR Unable to get apikey from firewall: local variable 'password' referenced before assignment ".

### ISSUE THREAD

**Issue by hamidbg86** [PaloAltoNetworks/Splunk-Apps#285]
## Describe the bug

I have properly setup the user in the Add-on, and still get this error. 

## Expected behavior

when I use the pantag command should send the ip-field to the dynamic group. 

## Current behavior

in splunk when you run a search like this : 
index=pan_logs  src_ip=publicip | pantag ip_field="src_ip" device="PanoramaIP" action="add" tag="ipblock"

I get the error: 
External search command 'pantag' returned error code 2. Script output = "ERROR Unable to get apikey from firewall: local variable 'password' referenced before assignment ".

## Possible solution

I test the api call to see if I can get the apikey and it was successful: 
curl -k -X POST  'https://panorama/api/?type=keygen&user=Account&password=password' -v

## Steps to reproduce

1. in splunk search run this command: index=pan_logs  src_ip=publicip | pantag ip_field="src_ip" device="PanoramaIP" action="add" tag="ipblock"
2.Notice the error : External search command 'pantag' returned error code 2. Script output = "ERROR Unable to get apikey from firewall: local variable 'password' referenced before assignment ".

## Context

basically trying to add src_ip to a dynamic group using the ipblock tag. 

## Your Environment

- Version used: 7.1.0
- Environment name and version (e.g. Chrome 59, node.js 5.4, python 3.7.3):
- Operating System and version (desktop or mobile): desktop
- Splunk Version 9.0.2

**Comment by hamidbg86** [comment]
@paulmnguyen  Please any idea about this issue?

**Comment by paulmnguyen** [comment]
@hamidbg86 Sorry I haven't had a chance to look at this one yet. I will see if I can reproduce it on my end and let you know what I find.

**Comment by paulmnguyen** [comment]
I wasn't able to reproduce this on my end using a fresh install. I am able to make the API calls. Could you try removing your passwords.conf and splunk_ta_paloalto_account.conf from your splunk_ta_paloalto/local directory
